Codigo/Protocol_GmCommands.bas

```diff
@@ -2035,6 +2035,7 @@ Public Sub HandleForgive(ByVal UserIndex As Integer)
                 Exit Sub
             Else
                 Call WriteChatOverHead(UserIndex, "Para volver a ser un ciudadano deberás donar 10.000 monedas de oro.", priest.Char.charindex, vbWhite)
+                Exit Sub
```

Comment:
with this change al code left in the function has no use and should be deleted, we need to think this a little more